backend/coins/btc/handlers/handlers.go

```diff
@@ -221,8 +221,8 @@ func (handlers *Handlers) getTxInfoJSON(txInfo *accounts.TransactionData, detail
 		Note:         handlers.account.TxNote(txInfo.InternalID),
 	}
 
+	txInfoJSON.Fee = feeString
 	if detail {
```

Comment:
I slightly prefer if we could just pass all that data anyways.

Reply:
interestingly TypeScript already expected that there is fee, also it thinks there is gas and nonce.  I am planing to open another PR to fix the types and make gas/nonce only be in /transaction?id= endpoint and optional.

https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/12df2de2626dec6e53970f4ddbed45cb888450d0/frontends/web/src/api/account.ts#L247-L250